File: backend/src/main/java/com/ll/hotel/domain/booking/booking/dto/BookingFormResponse.java
package com.ll.hotel.domain.booking.booking.dto;

import com.ll.hotel.domain.hotel.hotel.dto.HotelDetailDto;
import com.ll.hotel.domain.hotel.room.dto.RoomDto;
import com.ll.hotel.domain.member.member.dto.MemberDTO;

public record BookingFormResponse (
        HotelDetailDto hotel,
        RoomDto room,
        String[] thumbnailUrls,
        MemberDTO member
) {
}

File: backend/src/main/java/com/ll/hotel/domain/booking/booking/dto/BookingFormResponse.kt
package com.ll.hotel.domain.booking.booking.dto

import com.ll.hotel.domain.hotel.hotel.dto.HotelDetailDto
import com.ll.hotel.domain.hotel.room.dto.RoomDto
import com.ll.hotel.domain.member.member.dto.MemberDTO

data class BookingFormResponse(
    val hotel: HotelDetailDto,
    val room: RoomDto,
    val thumbnailUrls: List<String>,
    val member: MemberDTO
)